tests/by-util/test_ls.rs

```diff
@@ -3393,3 +3393,42 @@ fn test_tabsize_formatting() {
         .succeeds()
         .stdout_is("aaaaaaaa bbbb\ncccc     dddddddd");
 }
+
+#[cfg(any(
```

Comment:
isn't it just unix ? :)

Reply:
They contain most UNIX OSes, but there are exceptions like `haiku`, `Redox`